PhpAmqpLib/Message/AMQPMessage.php

```diff
@@ -39,9 +39,9 @@ class AMQPMessage extends GenericContent
 
     /**
      * @param string $body
-     * @param null $properties
+     * @param array $properties
      */
-    public function __construct($body = '', $properties = null)
+    public function __construct($body = '', $properties = array())
```

Comment:
You can't change $properties to default to an empty array.  Or else the `GenericContent::__construct` will take in an empty array and the line `if ($properties)` will incorrectly return true.